src/routes/(app)/c/private/[channel_id]/+page.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    //TEST: 066ed576-645f-46a3-b8ee-8b6429d233cf
+    import InfiniteScroll from '$lib/components/InfiniteScroll.svelte';
+
+    export let data;
+
+    let filter = 'all';
+    let sort = 'date';
+    let reverseSort = 'false';
+
+    const now = new Date();
+
+    const get_posts = async (page: number) => {
```

Comment:
`get_posts` should be `getPosts` (camelCase). We have linter changes [in another PR](https://github.com/Channel-Surfers/Channel-Surfers/pull/33) that will yell at us for this.